data/changelog/opam/2022-12-15-opam-2-1-4.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: opam 2.1.4 release
+title: opam 2.1.4, opam-publish 2.2.0, and opam-file-format 2.1.5 releases
```

Comment:
Oh, I didn't realise it was a release of several packages!

We're featuring the Platform tools on the Changelog, so I think we can omit `opam-file-format` in the title, and what do you think of splitting the announcement for `opam` and `opam-publish`? The advantage is that the changelog allows you to filter by tool, so you can have a list of releases for a specific tool

Reply:
Each one has its own changelog list, feel free to update. Other posts are in the same case, like [this one](https://opam.ocaml.org/blog/opam-2-0-10-2-1-1-depext/)
It will also ease filling `changelog` field.

There is also a [merlin post](https://opam.ocaml.org/blog/merlin-2-0-0-released/) that is in the opam directory too.

Reply:
Thanks, will update them!